demisto_sdk/tests/integration_tests/generate_docs_integration_test.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+from demisto_sdk.commands.common.git_tools import git_path
+from os.path import join
+from pathlib import Path
+
+from click.testing import CliRunner
+from demisto_sdk.__main__ import main
+
+GENERATE_DOCS_CMD = "generate-docs"
+DEMISTO_SDK_PATH = join(git_path(), "demisto_sdk")
+VALID_PLAYBOOK_WITH_IO = join(DEMISTO_SDK_PATH, "tests/test_files/playbook-Test_playbook.yml")
+VALID_PLAYBOOK_NO_IO = join(DEMISTO_SDK_PATH, "tests/test_files/Playbooks.playbook-test.yml")
+
+
+class TestPlaybooks():
+    def test_integration_generate_docs_playbook_positive_with_io(self, tmpdir):
+        """
+        Given
+        - Path to valid playbook yml file to generate docs for.
+        - Path to directory to write the README.md file.
+        - The playbook has inputs.
+        - The playbook has outputs.
+
+        When
+        - Running the generate-docs command.
+
+        Then
+        - Ensure README.md is created.
+        - Ensure README.md has an inputs section.
+        - Ensure README.md has an outputs section.
+        """
+
+        runner = CliRunner(mix_stderr=False)
+        arguments = [
+            GENERATE_DOCS_CMD,
+            '-i', VALID_PLAYBOOK_WITH_IO,
+            '-o', tmpdir
+        ]
+        _ = runner.invoke(main, arguments)
```

Comment:
also assert status code, no stderr, stdout and no exception

Reply:
I'm not checking `stdout` just cause it can output that you're not using the latest version, (if you're testing locally).